.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "FitNesse",
+  "image": "mcr.microsoft.com/vscode/devcontainers/base:focal",
+  "features": {
+    "ghcr.io/devcontainers/features/java:1": {
+      "version": "11",
+      "installMaven": "false",
+      "installGradle": "false"
```

Comment:
Given that this container just seems to get the source of the project I would expect it to get Gradle also, so it can compile the source.

Is that not needed for VSCode?